### DISCUSSION TITLE
Securityonion extremly slow

### DISCUSSION THREAD

**Discussion by cometsflyinginsect** [Security-Onion-Solutions/securityonion#9862] [Unsupported Versions]
I recently discovered SO and installed it a few days ago, leaving all settings at default in the installer. The features look realy cool but my instance is extremly slow making it unuseable (every click takes approx 3 minutes).
It's running in a virtualized environment based on VMWare and the virtual hard drive is located on an all-flash-storage. I assigned my machine a bunch of ressources, 48 GB RAM and 12 cores, but that doesn't help.

Do you have any ideas why it's so slow?

**Comment by tomjones1977** [comment]
It definitely sounds like there is an issue with your setup.  I've got virtualised SO instances for students to use with 20GB RAM and 8 cores and they're responsive.  Not snappy but definitely functional.

I'd take a look at the resources in use from a SO perspective, `top`, `htop` etc and see if you can see any obvious issues.  Check the IO wait time as well - all flash storage should be fine but worth looking at.
